src/components/SelectWithTitle.tsx

```diff
@@ -7,14 +8,14 @@ export interface SelectWithTitleProps {
 
 export default function SelectWithTitle({ title, selectProps }: SelectWithTitleProps) {
     return (
-        <>
-            {title}
+        <div className={'select-with-title-container'}>
+            <div className={'title'}>{title}</div>
             <Select
-                style={{ width: 120 }}
+                className={'select-element'}
                 // This always sets the first value as default value, but can be overwritten by setting defaultValue inside selectProps
                 defaultValue={selectProps?.options?.[0] ? selectProps?.options[0].value : null}
                 {...selectProps}
             />
-        </>
+        </div>
```

Comment:
We use the css classes from css modules as variables instead of strings to have type safety and auto complete. Please change it to always use the properties from the `styles` import

Reply:
Döner